src/modules/dashboard/components/CoinMachine/CoinMachine.css

```diff
@@ -19,7 +20,7 @@
 
 .grid {
   display: grid;
-  margin-top: 40px;
+  margin: 40px 0 20px;
```

Comment:
I think this 20px margin is not applied, I think it's useless here.

Reply:
Removed it